src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorIntegrationTest.java

```diff
@@ -1,342 +1,369 @@
 
-    @Test
-    public void GATK3LargeConcordanceTest() throws IOException {
-        assertVariantContextsMatch(getTestFile("HCOutput.NoAnnotations.vcf"), new File(getToolTestDataDir() + "expected/integrationTest.vcf"), Arrays.asList("-G", "Standard", "-G", "AS_Standard", "-L", "20:10000000-10100000", "-I", NA12878_20_21_WGS_bam), b37_reference_20_21);
-    }
+        // add the input as a comp -- every site shold get this annotation
+        final ArgumentsBuilder argsForTwoComps = new ArgumentsBuilder(argsForOneComp.getArgsArray())
+                .addFileArgument(StandardArgumentDefinitions.COMPARISON_LONG_NAME + ":" + FOO2, inputVCF);
 
-    @Test
-    public void testHasAnnotsNotAsking() throws IOException {
-        final File expected = new File(getToolTestDataDir() + "expected/testHsAnnotsNotAsking1.vcf");
-        final VCFHeader header = getHeaderFromFile(expected);
-        runVariantAnnotatorAndAssertSomething(getTestFile("vcfexamplemultisample.vcf"), new File(getToolTestDataDir() + "expected/testHsAnnotsNotAsking1.vcf"), Arrays.asList( "-I", largeFileTestDir + "CEUTrio.multisample.b37.1M-1M50k.bam"),
-                (a, e) -> {
-                    // We need to filter out sites where we saw a DP of 250 because we are comparing the results to GATK3, which downsamples to 250 reads per sample, which GATK4 does not currently support.
-                    if (!e.getGenotypes().stream().anyMatch(g -> g.hasDP() && g.getDP() >= 250)) {
-                        VariantContextTestUtils.assertVariantContextsAreEqualAlleleOrderIndependent(a, e, ATTRIBUTES_TO_IGNORE, header);
-                    }
-                },
-                b37_reference_20_21);
-    }
+        runCommandLine(argsForTwoComps.getArgsList());
 
-    @Test
-    public void testHasAnnotsAsking() throws IOException {
-        final File expected = new File(getToolTestDataDir() + "expected/testHasAnnotsAsking1.vcf");
-        final VCFHeader header = getHeaderFromFile(expected);
-        runVariantAnnotatorAndAssertSomething(getTestFile("vcfexamplemultisample.vcf"), new File(getToolTestDataDir() + "expected/testHasAnnotsAsking1.vcf"), Arrays.asList("-G", "Standard", "-I", largeFileTestDir + "CEUTrio.multisample.b37.1M-1M50k.bam"),
-                (a, e) -> {
-                    // We need to filter out sites where we saw a DP of 250 because we are comparing the results to GATK3, which downsamples to 250 reads per sample, which GATK4 does not currently support.
-                    if (e.getGenotypes().stream().noneMatch(g -> g.hasDP() && g.getDP() >= 250)) {
-                        VariantContextTestUtils.assertVariantContextsAreEqualAlleleOrderIndependent(a, e, ATTRIBUTES_TO_IGNORE, header);
-                    }
-                },
-                b37_reference_20_21);
-    }
-
-    @Test
-    public void testNoAnnotsNotAsking() throws IOException {
```

Comment:
I guess this one tests that if you don't supply any annotation args and the input has no annotations, then the output should have no annotations, which is pretty trivial.  Not sure I'm particularly concerned about testing that behavior.